source/resource/config/.templates/visu_config.xml

```diff
@@ -14,7 +14,7 @@
           - <a href=".?forceReload=true">Reload</a>
         ]]></status>
       <status type="html" condition="!edit" hrefextend="config"><![CDATA[
-          - <a href="editor/">Edit</a>
+          - <a href="#" onclick="qx.core.Init.getApplication().showManager('open', 'visu_config.xml')">Edit</a>
```

Comment:
The file name shouldn't be hard coded here, especially not for the template. This will lead to questions / bug reports when the user is creating a config with a different name

Reply:
Fixed